controllers/users.js

```diff
@@ -146,6 +149,40 @@ const postUserPicture = async (req, res) => {
   }
 };
 
+/**
+ * Updates the user data
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const updateUser = async (req, res) => {
+  try {
+    const userId = req.params.userId;
+    const { id: profileDiffId, message, ...profileDiff } = req.body;
+
+    const user = await userQuery.fetchUser({ userId });
+    if (!user.userExists) return res.boom.notFound("User doesn't exist");
+
+    const profileResponse = await profileDiffsQuery.update({ approval: "APPROVED" }, profileDiffId);
```

Comment:
Can we look into creating constants for strings?
Like `"Approved"`

Reply:
Ok. Will make the changes.